core/types/calls.cc

```diff
@@ -1363,6 +1363,26 @@ class Magic_buildArray : public IntrinsicMethod {
     }
 } Magic_buildArray;
 
+class Magic_buildRange : public IntrinsicMethod {
+public:
+    void apply(const GlobalState &gs, DispatchArgs args, const Type *thisType, DispatchResult &res) const override {
+        if (args.args.empty()) {
```

Comment:
Should this be `args.args.size() != 2`?

Reply:
The size can be `1` for `(x..)`. `0` should never happen as `(..)` is a parse error.

This was confusing, I changed it for `ENFORCE(!args.args.empty(), "Magic_buildRange called without argument");`